### DISCUSSION TITLE
Can you add automatice backups?

### DISCUSSION THREAD

**Discussion by markuswells** [FossifyOrg/General-Discussion#503] [Ideas]
I use Fossify Notes and it has an option for automatic backups. I am hoping it would be fairly easy to port that code into the messages app. I use syncthing to keep my notes backed up on my personal server and it would be great to be able to backup the messages also.

**Comment by Aga-C** [comment]
It's already requested: https://github.com/FossifyOrg/Messages/issues/1.
